tests/realtikvtest/addindextest/integration_test.go

```diff
@@ -144,12 +144,12 @@ func TestIngestMVIndexOnPartitionTable(t *testing.T) {
 	tk.MustExec("use addindexlit;")
```

Comment:
It is the biggest problem of ```addindextest``` that it is too big for this test package to complete in a short time. the ```timeout``` in the bazel config is ```long```. two component' timeout config is ```long``` in the tidb repo. I suggest we can split tests into new package such as ```pitr```.

Reply:
--- PASS: TestIngestMVIndexOnPartitionTable (452.41s)

The other tests in this package only take less than 1 min to finish.